<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsView</class>
 <widget class="QWidget" name="StatisticsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>统计报表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>报表类型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboReportType"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>开始日期:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateStart"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>结束日期:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEnd"/>
     </item>
     <item>
      <widget class="QPushButton" name="btGenerate">
       <property name="text">
        <string>生成报表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btExport">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>请选择报表类型并点击生成</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageText">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageChart">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="chartView" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTable">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>